src/view.tsx

```diff
@@ -0,0 +1,156 @@
+import { VDomRenderer } from '@jupyterlab/apputils';
+import { ThemeEditorModel } from './model';
+import React from 'react';
+import Form from '@rjsf/core';
+
+export class ThemeEditorView extends VDomRenderer<ThemeEditorModel> {
+  colorPicker = (props: any) => {
+    const { options } = props;
+    const { scope } = options;
+
+    return (
+      <input
+        type="color"
+        /*value={this.model.getColor(scope)}*/
+        onChange={event => {
+          this.model.setColor(scope, event.target.value);
+        }}
+      />
+    );
+  };
+
+  numberPicker = (props: any) => {
+    const { options } = props;
+    const { scope } = options;
+    return (
+      <input
+        type="number"
+        min="1"
+        max="40"
+        onChange={event => {
+          this.model.setNumber(scope, event.target.valueAsNumber);
+        }}
+      />
+    );
+  };
+
+  render(): JSX.Element {
+    return (
+      <Form
+        schema={{
```

Comment:
Later on it will be nice to generate the form directly in the model. Like that, you are sure to display variables that the model can handle.

Reply:
Thanks for the comment. I would like to implement that, tried but I don't know how this can be done.